src/mixins/controls.js

```diff
@@ -174,13 +212,15 @@ const OrbitControlsMixin = (superclass) => class extends superclass {
     }
 
     showSecondAxes() {
-        const secondAxes = [this.scene2.x, this.scene2.y, this.scene2.z].filter(x => x);  // assert no `undefined`;
-        this.scene2.add(...secondAxes);
+        this.scene2.secondAxesGroup = this.secondAxesGroup;
+        this.scene2.add(this.scene2.secondAxesGroup);
     }
 
     hideSecondAxes() {
-        const secondAxes = [this.scene2.x, this.scene2.y, this.scene2.z].filter(x => x);  // assert no `undefined`;
-        this.scene2.remove(...secondAxes);
+        this.scene2.remove(this.scene2.secondAxesGroup);
+        while (this.renderer2CSS2D.domElement.firstChild) {
+            this.renderer2CSS2D.domElement.removeChild(this.renderer2CSS2D.domElement.firstChild);
```

Comment:
DOM elements of labels are removed.

`secondAxesGroup` includes allows (`AllowHelper` object) and labels (`CSS2DObject`). When `AllowHelper` is removed, its DOM elements are automatically removed. But `CSS2DObject` is not. Therefore, we need to explicity remove DOM elements of labels.